elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/TableExportIT.java

```diff
@@ -0,0 +1,1441 @@
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.io.Files;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import lombok.Data;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.Principal;
+import java.util.Map;
+import java.util.stream.Stream;
+import java.util.stream.StreamSupport;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.SecurityContext;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class TableExportIT extends IntegrationTest {
+    private Integer port;
+
+    @Data
+    private class TableExport {
+        private String id;
+        private String query;
+
+        @JsonSerialize(using = EnumFieldSerializer.class, as = String.class)
+        private String queryType;
+        private Integer asyncAfterSeconds;
+        @JsonSerialize(using = EnumFieldSerializer.class, as = String.class)
+        private String resultType;
+
+        @JsonSerialize(using = EnumFieldSerializer.class, as = String.class)
+        private String status;
+    }
+
+    private static final Resource ENDERS_GAME = resource(
+            type("book"),
+            attributes(
+                    attr("title", "Ender's Game"),
+                    attr("genre", "Science Fiction"),
+                    attr("language", "English")
+            )
+    );
+
+    private static final Resource GAME_OF_THRONES = resource(
+            type("book"),
+            attributes(
+                    attr("title", "Song of Ice and Fire"),
+                    attr("genre", "Mythology Fiction"),
+                    attr("language", "English")
+            )
+    );
+
+    private static final Resource FOR_WHOM_THE_BELL_TOLLS = resource(
+            type("book"),
+            attributes(
+                    attr("title", "For Whom the Bell Tolls"),
+                    attr("genre", "Literary Fiction"),
+                    attr("language", "English")
+            )
+    );
+
+    public TableExportIT() {
+        super(AsyncIntegrationTestApplicationResourceConfig.class, JsonApiEndpoint.class.getPackage().getName());
+
+        this.port = super.getPort();
+    }
+
+    @Override
+    protected DataStoreTestHarness createHarness() {
+        DataStoreTestHarness dataStoreTestHarness = super.createHarness();
+        return new DataStoreTestHarness() {
+                public DataStore getDataStore() {
+                    return new AsyncDelayDataStore(dataStoreTestHarness.getDataStore(), 5000);
```

Comment:
5 seconds seems like a long time wait for each test.

Reply:
changing to 1 seconds